src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -3408,8 +3438,12 @@ subroutine cat_enkf_increments(                               &
 
     ! -------------------------------------------------------------------
     
-    implicit none
+    USE SurfParams, ONLY: WEMIN
+    USE StieglitzSnow, ONLY: RHOMA_pub, cpw_pub 
```

Comment:
The GEOSldas branch (feature/jpark50/MODIS_snow_cover_fraction_assimilation) points to the "develop" branch of the GEOSgcm_GridComp repository, which does not have "RHOMA_pub" and "cpw_pub".  I suspect that @jpark50 made the necessary changes to StieglitzSnow in his sandbox.  This will ultimately need to be resolved if the variables are needed.  (It doesn't look like RHOMA_pub is used.)